### ISSUE TITLE
a11y - mat-option text in the mat-select is not read by JAWS when DOWN_ARROW/ UP_ARROW is used to change option

### ISSUE THREAD

**Issue by VamsiDharmaji** [angular/material2#10683]
#### Bug, feature request, or proposal:
Bug verified on **JAWS Version** 2018.1712.10.400
**Chrome Version :** Version 65.0.3325.181 (Official Build) (64-bit)

#### What is the expected behavior?
JAWS should read mat-option text in the mat-select when DOWN_ARROW/ UP_ARROW is used to change option without using the ALT key to open the select options list popup

#### What is the current behavior?
JAWS doesn't read mat-option text in the mat-select when DOWN_ARROW/ UP_ARROW is used to change option without using the ALT key to open the select options list popup

#### What are the steps to reproduce?
StackBlitz : https://angular-agpwys.stackblitz.io/

1. Open JAWs
2. Open the URL in chrome
3. using Keyboard Focus ( use Tab key) into the select control
4. Use DOWN_ARROW/ UP_ARROW to change options

#### What is the use-case or motivation for changing an existing behavior?
We have a large user base using JAWS. We are migrating the application to angular and would like maintain the support for a11y

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 5.0
Angular Material 5.0

**Comment by jelbourn** [comment]
The a11y issues with select are being tracked in #5220 and #6382; we'll be revisiting select a11y in Q2
